DigitalLearningSolutions.Data.Tests/Services/SessionDataServiceTests.cs

```diff
@@ -139,13 +139,28 @@ public void UpdateSessionDuration_Should_Not_Update_Inactive_Session()
 
                 // When
                 const int sessionId = 468;
-                sessionDataService.UpdateSessionDuration(sessionId);
+                sessionDataService.UpdateDelegateSessionDuration(sessionId);
 
                 // Then
                 var updatedSessions = sessionTestHelper.GetCandidateSessions(candidateId);
 
                 updatedSessions.Should().BeEquivalentTo(startingSessions);
             }
         }
+
+        [Test]
+        public void StartAdminSession_Should_Start_Users_New_Admin_Session()
+        {
+            using (new TransactionScope())
+            {
+                // When
+                var adminUser = UserTestHelper.GetDefaultAdminUser();
+                var sessionId = sessionDataService.StartAdminSession(adminUser.Id);
+
+                // Then
+
+                sessionId.Should().NotBe(0);

```

Comment:
I think it might be worth doing something similar to the candidate tests where we fetch the session from the db (using the existing test helper) and check that admin id is associated with that particular session id. 
Also, super minor, but we can probably get rid of L161. 👀